src/configs/envConfig.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import { Alert } from 'react-native';
+import Intercom from 'react-native-intercom';
+import AsyncStorage from '@react-native-community/async-storage';
+import Storage from 'services/storage';
+import isEmpty from 'lodash.isempty';
+import RNRestart from 'react-native-restart';
+import { clearWebViewCookies } from 'utils/exchange';
+import { reportLog } from 'utils/common';
+import { firebaseIid } from 'services/firebase';
+
+const storage = Storage.getInstance('db');
+
+const buildType = __DEV__ ? 'development' : 'production';
+
+type CurrentEnvironment = {
+  [string]: string
+}
+
+// the following vars are CI/BUILD related and are fixed for both environment contexts
+const buildEnvironment = {
+  SENTRY_DSN: 'https://3ea39df26dd24e479c27642d11566e43@sentry.io/1294444',
+  BUILD_NUMBER: '_build_number_',
+  BUILD_TYPE: buildType,
+  OPEN_SEA_API_KEY: '_open_sea_api_key_',
+  INFURA_PROJECT_ID: '_infura_project_id_',
```

Comment:
What about infura project id, where should we put it now? Before it was in the env.development and it wasn't checked in the repo. Without it the app is constantly crashing.

Reply:
Just add it in your copy of configs/envConfig.js without committing the line.

Reply:
It would be very easy to commit it by accident though :/

Reply:
Hi, we discussed this with @IAmKio and came to the following commands (pull, change with your keys, then run these:)
 `git update-index --assume-unchanged src/configs/buildConfig.js`
 `git update-index --assume-unchanged .env`

It will temporary treat them as unchanged and will be out of the stack for committing